docs/actions/custom.md

```diff
@@ -27,20 +27,25 @@ For example, you could write an action that would update the `plan` column for a
 {% raw %}
 UPDATE invoices
 SET plan = {{ plan }}
-[[, payment = {{ payment }}]]
-WHERE id = {{ id }}
+   [[, payment = {{ payment }}]]
+WHERE
+   id = {{ id }}
 {% endraw %}
 ```
 
-The above code will create a form that prompts people to input updated values for the (required) `plan` field and optionally the `payment` field for a given record specified by `ID`. The code in brackets `[[ ]]` makes the statement optional: the bracket-enclosed statement will only run if someone inserts a value in the payment field.
+The above code will create a form that prompts people to input updated values for the (required) `plan` field and optionally the `payment` field for a given record specified by `ID`.
+
+![Example action form](./images/form.png)
+
+The code in brackets `[[ ]]` makes the statement optional: the bracket-enclosed statement will only run if someone inserts a value in the payment field. Note the comma that separates the statements is _inside_ the brackets.
```

Comment:
Can we move this note above the image? Since the note on brackets is more relevant to the code sample than the sample action form.